src/github.ts

```diff
@@ -281,10 +285,22 @@ export const cleanComments = async (
             params.page = params.page + 1;
             ctx.logger.debug(null, `cleanComments: increase page ${params.page}`);
           }
-          // save ids
-          botCommentIds.push(...response.data.filter((comment) => comment.user?.type === "Bot").map((r) => r.id));
+          const filteredComments = response.data
+            .filter((comment) => {
+              const isBot = comment.user?.type === "Bot";
+              // to avoid deleting the original comment
+              const isOriginComment = comment.id === originalComment.id;
```

Comment:
```suggestion
              const isOriginalComment = comment.id === originalComment.id;
```

Reply:
What is the `originalComment`? Is it the comment that includes `/bot clean --all`?